test/examples_test.go

```diff
@@ -291,10 +291,10 @@ func imageNamesMapping() map[string]string {
 			"registry":                              getTestImage(registryImage),
 			"node":                                  "node:alpine3.11",
 			"gcr.io/cloud-builders/git":             "alpine/git:latest",
-			"docker:dind":                           "ibmcom/docker-s390x:20.10",
+			"docker:dind":                           "icr.io/tekton-upstream/docker-s390x:20.10",
```

Comment:
NIT: these docker images are not owned and maintained by the Tekton community, so the `tekton-upstream` project name is slightly misleading.  I think it's fine to keep it, but perhaps a comment about the source / maintainer of the images would be nice

Reply:
Thanks, i will add maintainer details